src/ui/templates/results/verticalresults.hbs

```diff
@@ -34,9 +35,12 @@
           data-component="IconComponent"
           data-opts='{ "iconUrl": "{{ _config.icon }}" }'></div>
       {{/if}}
-      <div class="yxt-Results-titleLabel">{{_config.title}}</div>
+      <div class="yxt-Results-title">{{_config.title}}</div>
```

Comment:
it looks like in Yext this used to be an h2? is this change to a div intentional?

Reply:
I'll change this back to h2 in #873 (the following pr), and will check for unnecessary dom changes